src/components/checkbox/checkbox-group/checkbox-group.ts

```diff
@@ -1,5 +1,5 @@
 import type { CSSResultGroup, PropertyValues, TemplateResult } from 'lit';
-import { html, LitElement } from 'lit';
+import { LitElement, html } from 'lit';
```

Comment:
Why? :D

Reply:
VS Code import auto-format sometimes moves things around like this 😅